src/test/java/com/github/underscore/ObjectsTest.java

```diff
@@ -365,16 +365,16 @@ public void isError() {
     public void tap() {
         final List<Map.Entry<String, Integer>> result = new ArrayList<Map.Entry<String, Integer>>();
         $.tap((new LinkedHashMap<String, Integer>() { { put("a", 1); put("b", 2); put("c", 3); } }).entrySet(),
-            new Block<Map.Entry<String, Integer>>() {
-                public void apply(final Map.Entry<String, Integer> item) {
+            new Consumer<Map.Entry<String, Integer>>() {
+                public void accept(final Map.Entry<String, Integer> item) {
                     result.add(item);
                 }
             });
         assertEquals("[a=1, b=2, c=3]", result.toString());
         final List<Map.Entry<String, Integer>> resultChain = new ArrayList<Map.Entry<String, Integer>>();
         $.chain((new LinkedHashMap<String, Integer>() { { put("a", 1); put("b", 2); put("c", 3); } }).entrySet())
-            .tap(new Block<Map.Entry<String, Integer>>() {
-                public void apply(final Map.Entry<String, Integer> item) {
+            .tap(new Consumer<Map.Entry<String, Integer>>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.tap\(Consumer\) ignored, is this OK in com\.github\.underscore\.ObjectsTest\.tap\(\)